package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-posthtml",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
this not minor version for gulp-posthtml, change to path please.

Reply:
Done.